workspace.json

```diff
@@ -285,6 +285,11 @@
                 "glob": "**/*.json",
                 "output": "/"
               },
+              {
```

Comment:
This is necessary because of the `.` right?

Usually we name the file `__dot__babelrc` and then replace with `{ dot: '.' }`